include/internal/libspdm_lib_config.h

```diff
@@ -13,4 +13,55 @@
 #include LIBSPDM_CONFIG
 #endif
 
+#if defined(LIBSPDM_ENABLE_SET_CERTIFICATE_CAP) && \
+    !defined(LIBSPDM_ENABLE_CAPABILITY_SET_CERTIFICATE_CAP)
+#ifdef _MSC_VER
+#pragma message("LIBSPDM_ENABLE_SET_CERTIFICATE_CAP is deprecated. Use " \
+    "LIBSPDM_ENABLE_CAPABILITY_SET_CERTIFICATE_CAP instead. This warning will be removed in a " \
+    "future release.")
+#else
+#warning LIBSPDM_ENABLE_SET_CERTIFICATE_CAP is deprecated. Use \
+    LIBSPDM_ENABLE_CAPABILITY_SET_CERTIFICATE_CAP instead. This warning will be removed in a \
+    future release.
+#endif /* _MSC_VER */
+#endif /* defined(LIBSPDM_ENABLE_SET_CERTIFICATE_CAP) */
+
+#if defined(LIBSPDM_ENABLE_CHUNK_CAP) && !defined(LIBSPDM_ENABLE_CAPABILITY_CHUNK_CAP)
+#ifdef _MSC_VER
+#pragma message("LIBSPDM_ENABLE_CHUNK_CAP is deprecated. Use LIBSPDM_ENABLE_CAPABILITY_CHUNK_CAP " \
+    "instead. This warning will be removed in a future release.")
+#else
+#warning LIBSPDM_ENABLE_CHUNK_CAP is deprecated. Use LIBSPDM_ENABLE_CAPABILITY_CHUNK_CAP \
+    instead. This warning will be removed in a future release.
+#endif /* _MSC_VER */
+#endif /* defined(LIBSPDM_ENABLE_CHUNK_CAP) */
+
+#if defined(MDEPKG_NDEBUG) && !defined(LIBSPDM_DEBUG_ENABLE)
+#ifdef _MSC_VER
+#pragma message("MDEPKG_NDEBUG is deprecated. Use LIBSPDM_DEBUG_ENABLE " \
+    "instead. This warning will be removed in a future release.")
+#else
+#warning MDEPKG_NDEBUG is deprecated. Use LIBSPDM_DEBUG_ENABLE \
+    instead. This warning will be removed in a future release.
+#endif /* _MSC_VER */
+#endif /* defined(MDEPKG_NDEBUG) */
+
+#if defined(LIBSPDM_DEBUG_ENABLE)
+#undef LIBSPDM_DEBUG_ASSERT_ENABLE
+#undef LIBSPDM_DEBUG_PRINT_ENABLE
+#undef LIBSPDM_DEBUG_BLOCK_ENABLE
+
+#define LIBSPDM_DEBUG_ASSERT_ENABLE (LIBSPDM_DEBUG_ENABLE)
+#define LIBSPDM_DEBUG_PRINT_ENABLE (LIBSPDM_DEBUG_ENABLE)
+#define LIBSPDM_DEBUG_BLOCK_ENABLE (LIBSPDM_DEBUG_ENABLE)
+#elif defined(MDEPKG_NDEBUG)
+#undef LIBSPDM_DEBUG_ASSERT_ENABLE
+#undef LIBSPDM_DEBUG_PRINT_ENABLE
+#undef LIBSPDM_DEBUG_BLOCK_ENABLE
+
+#define LIBSPDM_DEBUG_ASSERT_ENABLE 0
+#define LIBSPDM_DEBUG_PRINT_ENABLE 0
+#define LIBSPDM_DEBUG_BLOCK_ENABLE 0
```

Comment:
Should we maybe explicitly print out in the make process that debug config is enabled or disabled? Especially if we're potentially overriding user defines here

Reply:
I'll add more documentation to `spdm_lib_config.h`. When CMake is set to `DTARGET=Release` then all debug will be disabled, regardless of the value of `LIBSPDM_DEBUG_ENABLE` in `spdm_lib_config.h`.